LoginPage.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Formik, Field, Form } from 'formik';
+import { useHistory } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const LoginPage = () => {
+  const history = useHistory();
+
+  return (
+    <div className="container-fluid">
```

Comment:
Consider separating the UI and the business logic. You can create a separate service for handling the login logic.